platform/podjs/src/browserPod.ts

```diff
@@ -92,11 +121,15 @@ class StorePolyIn implements PolyIn {
     }
 
```

Comment:
It would be great to add the update and query interface here as well

Reply:
There you go: https://github.com/polypoly-eu/polyPod/pull/1117